dts/bindings/ospi/st,stm32-ospi.yaml

```
@@ -33,3 +36,33 @@ properties:
 
     pinctrl-names:
       required: true
+
+    dmas:
+      description: |
+        Optional DMA channel specifier, required for DMA transactions.
+        For example dmas for TX/RX on OSPI
+           dmas = <&dma1 5 41 0x10000>;
+
+        With, in each cell of the dmas specificier:
```

specificier -> specifier

---

fixed

---

The original error is present.
```suggestion
        With, in each cell of the dmas specifier:
```